src/OrchardCore.Modules/OrchardCore.Autoroute/Models/AutoroutePartExtensions.cs

```diff
@@ -14,16 +15,26 @@ public static IEnumerable<ValidationResult> ValidatePathFieldValue(this Autorout
                 yield return new ValidationResult(S["Your permalink can't be set to the homepage, please use the homepage option instead."], new[] { nameof(autoroute.Path) });
             }
 
-            if (autoroute.Path?.IndexOfAny(AutoroutePart.InvalidCharactersForPath) > -1 || autoroute.Path?.IndexOf(' ') > -1 || autoroute.Path?.IndexOf("//") > -1)
+            if (HasInvalidCharacters(autoroute.Path))
             {
                 var invalidCharactersForMessage = string.Join(", ", AutoroutePart.InvalidCharactersForPath.Select(c => $"\"{c}\""));
-                yield return new ValidationResult(S["Please do not use any of the following characters in your permalink: {0}. No spaces, or consecutive slashes, are allowed (please use dashes or underscores instead).", invalidCharactersForMessage], new[] { nameof(autoroute.Path) });
+                const string errorMessage = "Please do not use any of the following characters in your permalink: {0}. No spaces, or consecutive slashes, are allowed (please use dashes or underscores instead).";
+                yield return new ValidationResult(S[errorMessage, invalidCharactersForMessage], new[] { nameof(autoroute.Path) });
             }
 
             if (autoroute.Path?.Length > AutoroutePart.MaxPathLength)
             {
                 yield return new ValidationResult(S["Your permalink is too long. The permalink can only be up to {0} characters.", AutoroutePart.MaxPathLength], new[] { nameof(autoroute.Path) });
             }
         }
+
+        private static bool HasInvalidCharacters(string path)
+        {
+            if (path?.IndexOfAny(AutoroutePart.InvalidCharactersForPath) > -1) return true; // IndexOfAny performs culture-insensitive search.
```

Comment:
```suggestion
            // IndexOfAny performs culture-insensitive search.
            if (path?.IndexOfAny(AutoroutePart.InvalidCharactersForPath) > -1) 
            {
                return true;
            }
```

Reply:
@deanmarcussen code for early exit I prefer to use if without a curry brace. 
If without a curry brace has some drawback but I think it is okay for this situation.
However, as your suggestion, I will change.